database/migrations/2022_01_18_024625_create_posts_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreatePostsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('posts', function (Blueprint $table) {
+            $table->increments("id");
```

Comment:
```suggestion
            $table->increments('id');
```
mỗi dòng nháy 1 kiểu thế

Reply:
Vâng ạ, em sửa lại ngay